docs/CodeDevelopment/requirements.md

```diff
@@ -8,7 +8,29 @@
 4. Make sure it has docs.  Even a minimal readme.md will get collected and added to the docs.
 5. Make sure it has only valid characters encoded (often copy paste from Microsoft Word docs or the internet will lead
    to invalid characters)
-6. If it is a small change/tweak to existing code that originates outside of Project Mu please mark it with //MU_CHANGE
+6. Any changes made to code that originates from outside Project Mu (e.g. upstream project TianoCore, OpenSSL, etc.)
+   should be surrounded with comments/tags indicating that it is a "MU_CHANGE".
+    - Most of the contents of `mu_basecore`, `mu_tiano_plus`, `mu_silicon_arm_tiano`, and `mu_silicon_intel_tiano`
+      fall under this requirement.
+    - Example:
+
+    ```c
+    EFI_STATUS
+    EFIAPI
+    SomeFunction (
+            VOID
+            )
+        {
+            EFI_STATUS      Status;
+            // MU_CHANGE [BEGIN] - Add new counters for Feature X
```

Comment:
Is the single line comment/tag still allowed? If so, it might help clarify by providing a single line example as well.